### ISSUE TITLE
3.4.1 won't install via Carthage ("xcodebuild timed out while trying to read Lottie.xcworkspace")

### ISSUE THREAD

**Issue by stevelandeyasana** [airbnb/lottie-ios#1709]
## Which Version of Lottie are you using?

Lottie 3.4.1

## Expected Behavior 

Carthage install works

## Actual Behavior

```
mint run carthage update --platform iOS --use-xcframeworks --use-ssh --no-use-binaries --cache-builds
...
*** Fetching lottie-ios
...
*** Checking out lottie-ios at "3.4.1"
...
*** xcodebuild output can be found in /var/folders/9r/rlljjbhx6sngfnfbdrsdt6rr0000gq/T/carthage-xcodebuild.BitVMM.log
...
*** Invalid cache found for lottie-ios, rebuilding with all downstream dependencies
...
xcodebuild timed out while trying to read Lottie.xcworkspace 😭
```

This issue does not occur on 3.4.0.

**Comment by calda** [comment]
Could you share a sample project that reproduces this issue?

**Comment by stevelandeyasana** [comment]
I'll consider finding the time in the future. But this is purely running in Carthage, not building my project, so I'm not sure it would really be helpful.

Some googling suggests that if a project has SPM dependencies (which [lottie does](https://github.com/airbnb/lottie-ios/blob/master/Lottie.xcworkspace/xcshareddata/swiftpm/Package.resolved)) and the user has a slow internet connection (which I do), Carthage can time out due to xcodebuild needing to fetch the SPM dependencies. https://github.com/realm/realm-swift/issues/6549

**Comment by calda** [comment]
Does this still happen if you point carthage to `master` instead of 3.4.1? We briefly had a large number of transitive dependencies in our `Package.resolved` which are present in 3.4.1 but no longer present on `master`. That change should reduce the number of repos that need to be cloned, which may fix this issue.

**Comment by rachit-anurag** [comment]
This issue is still reproducible for Xcode 14.1 with Carthage 0.38.0 for the Lottie 3.4.1
